src-docs/src/views/tables/mobile/props_info.js

```diff
@@ -38,6 +38,12 @@ export const propsInfo = {
           required: false,
           type: { name: 'bool' },
         },
+        width: {
+          description:
+            'Allocates value given of the width of the container in mobile view (typically cells are contained to 50%)',
```

Comment:
👍 Nice. Just a little wordsmithing 

```suggestion
            'Applies the value to the width of the cell in mobile view (typically 50%)',
```